pages/report.vue

```diff
@@ -0,0 +1,303 @@
+<template>
```

Comment:
Correct me if I'm wrong, but I don't think that knossos properly handles `Error`, afaik you need to use `addNotification` in every case